.github/workflows/release.yaml

```
@@ -3,51 +3,114 @@
-
-                # pushing the helm charts
-                echo "Pushing Chart: $CHART_NAME"
-                echo "Version: $CHART_NAME-$CHART_VERSION.tgz"
-                helm push $CHART_NAME-$CHART_VERSION.tgz oci://${{ env.HELM_CHART_REPOSITORY}}
-                echo "Successfully pushed chart: $CHART_NAME"
-            done
+          cd charts
+          CHART_DIR=elasti
+          CHART_NAME=${{ needs.read-chart-values.outputs.chart_name }}
+          CHART_VERSION=${{ needs.read-chart-values.outputs.chart_version }}
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)